tests/pgconsul.featureset

```diff
@@ -1,27 +1 @@
-features/pgconsul_util.feature
-features/initialization.feature
-features/priority.feature
-features/async.feature
-features/kill_primary.feature
-features/kill_replica.feature
-features/kill_non_ha_replica.feature
-features/slot.feature
-features/coordinator.feature
-features/coordinator_fail.feature
-features/single_node.feature
-features/op_track.feature
 features/maintenance.feature
```

Comment:
for faster debug in github actions

Reply:
> for faster debug in github actions

Actually it is not used in github, we have matrix strategy, launching every feature in separate test: https://github.com/yandex/pgconsul/blob/main/.github/workflows/docker-behave.yml#L14

Reply:
Oh... thanks